awkward/pandas/array/jagged.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+
+# BSD 3-Clause License; see https://github.com/scikit-hep/awkward-array/blob/master/LICENSE
+
+import numpy
+
+import awkward.array.jagged
+from pandas.api.extensions import ExtensionArray
+from awkward.pandas.accessor import AwkwardType
+
+class JaggedArrayPandas(awkward.array.jagged.JaggedArray, ExtensionArray):
+
+    @property
+    def name(self):
+        return str('awkward')
+
+    @property
+    def dtype(self):
+        return AwkwardType()
```

Comment:
I suppose all the awkward classes should have `property dtype == AwkwardType()`? And the `name` as well?

Reply:
This is overriding what `dtype` was supposed to mean, but that's Pandas's fault.

Reply:
Yes, they need to have that custom `dtype` (as far as I can tell).

I just checked, and it turns out that the `name` property is not needed in the way that I thought it was, so it can be removed.